melkor/datasets/ames_dataset.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import requests
+from datetime import datetime
+from pathlib import Path
+from melkor.base import PandasBaseDataset
+from melkor.utils import to_snake
+
+
+class AmesDataset(PandasBaseDataset):
+
+    URL = "https://katrienantonio.github.io/hands-on-machine-learning-R-module-1/data/ames_python.csv"
```

Comment:
This is what I meant before , URL and filename are in the code code, this can change at any time, and is incovenient having to change the python code for it